timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -146,7 +146,7 @@ class AbstractGluonTSModel(AbstractTimeSeriesModel):
 
     gluonts_model_path = "gluon_ts"
     # default number of samples for prediction
-    default_num_samples: int = 1000
+    default_num_samples: int = 250
```

Comment:
This parameter is only used by a single model (DeepAR), where it is overridden to 250. We might as well keep the overridden value here, as no other model generates SampleForecast output.